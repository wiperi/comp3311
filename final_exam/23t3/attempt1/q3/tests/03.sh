psql funrun -c 'select * from q3(9);'
