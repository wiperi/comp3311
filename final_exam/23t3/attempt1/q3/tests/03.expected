                       q3                       
------------------------------------------------
 John Ng gave up at Yarra Boulevard
 Leebien Cardassilaris gave up at Bellbird Park
(2 rows)
